src/main/java/org/kie/processmigration/service/impl/MigrationServiceImpl.java

```diff
@@ -145,7 +125,7 @@ public Migration update(Long id, MigrationDefinition definition) throws Migratio
             throw new ReScheduleException("The migration execution type MUST be ASYNC");
         }
         migration.setDefinition(definition);
-        txHelper.withTransaction(() -> em.merge(migration));
+        txHelper.withTransaction((Runnable) migration::persist);
```

Comment:
Maybe we should call a merge operation, just in case
```suggestion
        txHelper.withTransaction(() -> Panache.getEntityManager().merge(migration));
```